mysql-test/lib/My/SafeProcess/wsrep_check_version.c

```diff
@@ -0,0 +1,123 @@
+/* Copyright (c) 2009, 2019, MariaDB
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; version 2 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02111-1301 USA
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#ifdef _WIN32
```

Comment:
Galera is not supported on Windows so this is totally unnecessary.

Reply:
I propose to leave this small code fragment - just in case, it will help if someone compiles the windows version with WITH_WSREP not through the standard configuration, but manually. Or if the Galera support policy for the Windows environment will be changed (and Galera will be supported there).